consensus/blake3pow/consensus.go

```diff
@@ -344,6 +353,42 @@ func (blake3pow *Blake3pow) CalcDifficulty(chain consensus.ChainHeaderReader, pa
 	return parent.Difficulty()
 }
 
+func (blake3pow *Blake3pow) CalcEntropyThreshold(chain consensus.ChainHeaderReader, parent *types.Header) *big.Int {
```

Comment:
I wonder if this is correct. This seems like 'top down' decision making -- doms projecting their thresholds onto the subs. My intuition is that the system is already fully controlled via the DAA, and the thresholds for coincidence should just be simple multiples of that.

Perhaps: `entropy_threshold = 30x difficulty @ prior coincident from that sub`

Reply:
Quai Network offers high-speed block production by running projects in cooperation with the 13 blockchain. To understand the high speed offered in block generation, if we compare it with other chain structures, for example, Prime chain's approximate block time is 15 minutes, Region chain's approximate block time is 5 minutes, and Zone chain's approximate block time is 10 seconds. While all these chains generate blocks asynchronously, the average block time of Quai Network is about 1 minute.